ui/src/components/Views/Configs/SettingsForm.tsx

```diff
@@ -175,8 +175,8 @@ export const SettingsForm = ({ initialState }: MountPointFormProps): ReactElemen
           {
             activeStep === 1 &&
             <Typography>
-              In order to start the Pro container, add a configuration with the variable LOCALSTACK_API_KEY 
-              set to your API key and select that configuration in the top right corner
+              In order to start the Pro container, add a configuration with the variable LOCALSTACK_AUTH_TOKEN 
+              set to your auth token and select that configuration in the top right corner
```

Comment:
we are still supporting auth tokens, so this should probably mentioned API Keys as well, i.e.

`API Keys are also supported, but will be deprecated in the future`